js/hoot/view/utilities/Utilities.js

```diff
@@ -173,34 +173,15 @@ Hoot.view.utilities = function (context){
         var labelContainer = header.append('div');
 
         labelContainer.append('div')
+        .classed('point', true)
         .attr('href', '#version')
         .append('img')
-        .classed('col1 cursor', true)
-        .attr('src', 'img/hoot_logo_icon.png')
-        .attr('height', '60px');
-        labelContainer.append('div')
-        .classed('inline dark strong hoot_label cursor col2', true)
-        .html('Hootenanny')
+        .attr('src', context.asset('dist/img/logo/hoot_logo_update.png'))
```

Comment:
I don't think you want the 'dist' in the path here.  That's what context.asset( ) adds if necessary.

Does this work when deployed via hootserver and tomcat?